client/src/components/Form/Form.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export const Form = ({onSubmit, inputs, buttonText}) => {
```

Comment:
let's make a habit of keeping our objects and object destructuring alphabetical
the benefit being that when our objects or destructures get very complicated, it's easier for us to be able to expect where a property should be rather than needing to read over each element

Reply:
thanks for the reminder! I will update the objects / destructuring lines and push.